spec/features/user_views_items_index_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+feature 'user visits items index', %{
+  As a visitor
+  I want to visit the items index
+  So that I can see items specific to a particular category
+} do
+
+  # Acceptance Criteria:
+  # * Visitor must see list of relevant items
+  # * Visitor must not see items of another category
+
+  scenario 'user visits root path' do
+    item1 = FactoryGirl.create(:item, category_id: 1)
```

Comment:
If you make the change below to your factory you will not have to add in the category id hard coded here.